chore(deps): bump aws-cdk from 2.54.0 to 2.70.0

joel-wong-aws opened the pull request:
Bumps [aws-cdk](https://github.com/aws/aws-cdk/tree/HEAD/packages/aws-cdk) from 2.54.0 to 2.70.0.
- [Release notes](https://github.com/aws/aws-cdk/releases)
- [Changelog](https://github.com/aws/aws-cdk/blob/main/CHANGELOG.v2.md)
- [Commits](https://github.com/aws/aws-cdk/commits/v2.70.0/packages/aws-cdk)

---
updated-dependencies:
- dependency-name: aws-cdk dependency-type: direct:development update-type: version-update:semver-minor ...


----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

ddneilson commented on CHANGELOG.md:
```diff
@@ -7,7 +7,7 @@ All notable changes to this project will be documented in this file. See [standa
 ### Supported CDK Version
 
-* [2.54.0](https://github.com/aws/aws-cdk/releases/tag/v2.54.0)
+* [2.69.0](https://github.com/aws/aws-cdk/releases/tag/v2.69.0)
```
This shouldn't be changed. It's automatically generated when we do the release.

joel-wong-aws replied:
The change has now been removed!

joel-wong-aws committed:
chore(deps): bump aws-cdk from 2.54.0 to 2.70.0

Bumps [aws-cdk](https://github.com/aws/aws-cdk/tree/HEAD/packages/aws-cdk) from 2.54.0 to 2.70.0.
- [Release notes](https://github.com/aws/aws-cdk/releases)
- [Changelog](https://github.com/aws/aws-cdk/blob/main/CHANGELOG.v2.md)
- [Commits](https://github.com/aws/aws-cdk/commits/v2.70.0/packages/aws-cdk)

---
updated-dependencies:
- dependency-name: aws-cdk
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

joel-wong-aws commented:
CDK version 2.70.0 has been released! Updated the pull request accordingly. Leaving the branch name as-is to keep the pull request comment history.
